cvFit/src/app/workoutList/workoutlist.component.html

```diff
@@ -9,6 +9,11 @@ <h1>
   </h1>
   <hr>
   <div class="flex-container" *ngIf='workouts$ | async as workouts'>
+    <div *ngIf="!workouts?.length">
+      <h1>No workouts to display!</h1>
```

Comment:
Don't use header tags just for styling. They're meant to define a hierarchy for your page. You should not have more than one `h1` on a page. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements